scripts/install-tools.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash 
+# Copyright © 2025 Apple Inc. and the container-builder-shim project authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GOLANGCI_LINTER_VERSION=v2.1.6
+
+install_golangci() {
+  ci_install=false
+  if [[ -x $(which golangci-lint) ]]; then
+      local version=$(golangci-lint version --short)
+      if [[ "${version}" == "${GOLANGCI_LINTER_VERSION#v}" ]]; then
+          ci_install=false
+        else
+          ci_install=true
+      fi
+    else
+      ci_install=true
+  fi
+  if [[ "$ci_install" == true ]]; then
+      curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/cc3567e3127d8530afb69be1b7bd20ba9ebcc7c1/install.sh \
```

Comment:
In the other container project repos, we tend to install things under a .local directory for make to use (example https://github.com/apple/container/blob/9eb2f860fc35a933e7894bc5313e3b79cd001c52/Makefile#L162).